cmd/soroban-rpc/internal/methods/simulate_transaction.go

```diff
@@ -114,21 +120,32 @@ func NewSimulateTransactionHandler(logger *log.Entry, ledgerEntryReader db.Ledge
 			}
 		}
 
+		var results []SimulateHostFunctionResult
+		if result.Result != "" {
```

Comment:
oh - you're omitting the `Results` in case it's empty, as per the response schema.
That's perfectly fine. @Shaptic just wanted to know you're aware of that.

Reply:
Yep, that's totally fine; we already assume this field is optional: https://github.com/stellar/js-soroban-client/blob/main/src/soroban_rpc.ts#L132-L133.

Reply:
Great